packages/s3-cache/src/index.ts

```diff
@@ -23,12 +23,17 @@ export class S3Cache implements CacheStrategy {
   async get(pageKey: string, cacheKey: string): Promise<CacheEntry | null> {
     if (!this.client) return null
 
-    const pageData = await this.client.getObject({
-      Bucket: this.bucketName,
-      Key: `${this.removeSlashFromStart(pageKey)}/${cacheKey}.json`
-    })
+    const pageData = await this.client
+      .getObject({
+        Bucket: this.bucketName,
+        Key: `${this.removeSlashFromStart(pageKey)}/${cacheKey}.json`
+      })
+      .catch((error) => {
```

Comment:
could you explain more why do we need that change? it was done intentionally so logger can catch that here https://github.com/DBB-Software/next-cache-handler/blob/main-v2/packages/cache-core/src/cacheHandler.ts#L132 and write data to logs, now we are going to miss that

Reply:
We will miss only the **NotFound** and **NoSuchKey** types of error. 
I think it is not an error and is more relater to the cases when we don't have an actual cache. 
Also, it is consistent with the behavior of others' strategies when we recognize these cases as having no actual cache. 

If we don't handle these cases, We will have errors during the build time(when we start from scratch), after deleting the cache for example it can be revalidating by tag/path or TTL

Reply:
yes, but it is still useful for debugging if your app can not find cache key for some reason imo

+ if we still want to improve it we should log info module in such case without throwing an error OR make a custom error which should that just data for that key was not found

Reply:
Thank you for clarifying, if we had a small sync about it and other things, it would be nice and we would find the best solution. Let's discuss it tomorrow. Okay?

Reply:
sure, mu suggestion would be:
1. keep logic for not found here
2. update cache handle to use  logger `info` and track when there is no data for given page https://github.com/DBB-Software/next-cache-handler/blob/main-v2/packages/cache-core/src/cacheHandler.ts#L132

Reply:
Updated with your suggestion